lib/warder.rb

```diff
@@ -4,7 +4,7 @@
 require 'warder/style_guide_runner'
 require 'warder/magick_numbers_runner'
 require 'warder/code_duplication_runner'
-require 'warder/code_smells_runner'
+require 'warder/code_smell_runner'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.